java/common/src/main/java/com/exadel/frs/commonservice/exception/FaceNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.exadel.frs.commonservice.exception;
+
+import static com.exadel.frs.commonservice.handler.CommonExceptionCode.FACE_NOT_FOUND;
+
+public class FaceNotFoundException extends BasicException {
+
+    private static final String MESSAGE = "Face %s not found";
```

Comment:
Let's return "Subject %s not found" error

Reply:
Done